cassovary-core/src/main/scala/com/twitter/cassovary/util/io/ListOfEdgesGraphReader.scala

```diff
@@ -112,7 +112,7 @@ class ListOfEdgesGraphReader[T](
 
       val edgesBySource = readEdgesBySource()
 
-      private lazy val edgesIterator = edgesBySource.entrySet().iterator()
+      private lazy val edgesIterator = edgesBySource.int2ObjectEntrySet().fastIterator()
```

Comment:
No, I didn't do any benchmarks but is recommended by documentation on fastutils.